lib/actions/ssh.ts

```diff
@@ -13,13 +13,15 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+import { BalenaDeviceNotFound } from 'balena-errors';
 import * as BalenaSdk from 'balena-sdk';
 import { CommandDefinition } from 'capitano';
 import { stripIndent } from 'common-tags';
 
-import { BalenaDeviceNotFound } from 'balena-errors';
 import { validateDotLocalUrl, validateIPAddress } from '../utils/validation';
 
+const MIN_DIRECT_SSH_VERSION = '2.44.0';
```

Comment:
Perhaps I should know the answer, but if you would indulge me, :-) what is it that 2.44 has to allow "direct ssh"?

Reply:
As of 2.44.0 BalenaOS is running opensshd which allows us to authenticate against ssh keys in balena-api, meaning there is no necessity to go via balena-proxy (gateway).